newIDE/app/src/ResourcesList/index.js

```diff
@@ -291,12 +291,23 @@ export default class ResourcesList extends React.Component<Props, State> {
       { type: 'separator' },
       {
         label: i18n._(t`Remove unused...`),
-        submenu: allResourceKindsAndMetadata.map(({ displayName, kind }) => ({
-          label: i18n._(displayName),
-          click: () => {
-            this.props.onRemoveUnusedResources(kind);
-          },
-        })),
+        submenu: allResourceKindsAndMetadata
+          .map(({ displayName, kind }) => ({
+            label: i18n._(displayName),
+            click: () => {
+              this.props.onRemoveUnusedResources(kind);
+            },
+          }))
+          .concat([
+            {
+              label: i18n._('All resources'),
+              click: () => {
+                allResourceKindsAndMetadata.forEach(resource => {
```

Comment:
The variable name `resource` does not really represents a resource here. Think about what it really is:
it seems to be a "resourceKindAndMetadata", according to the name of the thing you iterate on.
You can also take the approach of "not naming this thing" and destructure it directly (`{ kind }`).